src/main/java/net/starlark/java/eval/Starlark.java

```diff
@@ -206,6 +206,8 @@ public static Object fromJava(Object x, @Nullable Mutability mutability) {
       return StarlarkList.copyOf(mutability, (List<?>) x);
     } else if (x instanceof Map) {
       return Dict.copyOf(mutability, (Map<?, ?>) x);
+    } else if (x instanceof Set && ((Set<?>) x).size() == 1) {
```

Comment:
Why is this needed? We are usually more thorough on reviews of Starlark changes. If you need it, perhaps it's better to submit it in a separate PR.